README.md

```diff
@@ -125,4 +125,4 @@ Feel free to check the [issues page](https://github.com/Olivier-Kango/vet_clinic
 This project is [MIT](./LICENSE.txt) licensed.
 
 
```

Comment:
Great job @Olivier-Kango 🤝

- [ ] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.

Reply:
@1GORDON Thanks a lot I fix it right now!